### PR TITLE
Touching up the April Activity Report

### PR DESCRIPTION
**Pull request by CodeforEvolution**
Just some fixes for some grammar and vocab errors. It happens to the best of us. :)

### CODE REVIEW

File: content/blog/pulkomandy/2020-04-28-haiku_activity_report_april_2020.md [hunk 1]
```diff
@@ -25,7 +25,7 @@ next one!
 <h3>Outside the source tree</h3>
 
 To celebrate the new beta release, 3dEyes has been working on porting GTK
-to Haiku. While of course our main focus remain on native applications using the
+to Haiku. While of course our main focus remains on native applications using the
 BeAPI, it's still a great help for new users if they can find some of their well
 known apps at first. The port is nowhere near ready yet, but we've seen some
 screenshots already...
```
Comment (humdingerb) [inline content/blog/pulkomandy/2020-04-28-haiku_activity_report_april_2020.md:31]: Our forum software seems to have an issue display elipses. It shows "& mldr;" instead of the dots. Better use 3 dots "..." instead.

File: content/blog/pulkomandy/2020-04-28-haiku_activity_report_april_2020.md [hunk 2]
```diff
@@ -70,66 +70,66 @@ Korli worked on various low-level drivers: for PCI thermal sensors, for the I2C
...
-BeOS), but in other cases we decided to follow what BeOS did.
+BeOS), but in other cases, we decided to follow what BeOS did.
 
 <h3>Interface Kit</h3>
 
-Support for "transparent" views was added. This allow a view to let its parent
+Support for "transparent" views was added. This allows for a view to let its parent
 draw itself, allowing, as the name implies, for some transparency effects. This
-is used in various places (replicant handles, ...) to fix drwaing glitches.
+is used in various places (replicant handles, ...) to fix drawing glitches.
```
Comment (humdingerb) [inline content/blog/pulkomandy/2020-04-28-haiku_activity_report_april_2020.md:125]: Again elipsis -> "..."

File: content/blog/pulkomandy/2020-04-28-haiku_activity_report_april_2020.md [hunk 3]
```diff
@@ -70,66 +70,66 @@ Korli worked on various low-level drivers: for PCI thermal sensors, for the I2C
...
+showed that the current default settings are mostly fine, except the
+"Tracker always first" option, which is now enabled by default.
 
 <h3>Unit Tests</h3>
 
 Kyle Ambroff-Kao continues his work on getting the unit tests to run, with
-fixes in the network and storage kit tests. These led to discovery of some
-edge cases where Haiku behavior was different from BeOS, in some, the new
+fixes in the Network and Storage Kit tests. These led to the discovery of some
+edge cases where Haiku behavior was different from BeOS. In some, the new
```
Comment (humdingerb) [inline content/blog/pulkomandy/2020-04-28-haiku_activity_report_april_2020.md:117]: Maybe even better:
```suggestion
edge cases where Haiku behavior was different from BeOS. In some cases, the new
```
[plus, see next comment]

File: content/blog/pulkomandy/2020-04-28-haiku_activity_report_april_2020.md [hunk 4]
```diff
@@ -70,66 +70,66 @@ Korli worked on various low-level drivers: for PCI thermal sensors, for the I2C
...
 behavior is obviously better (we don't aim for bug-to-bug compatibility with
-BeOS), but in other cases we decided to follow what BeOS did.
+BeOS), but in other cases, we decided to follow what BeOS did.
```
Comment (humdingerb) [inline content/blog/pulkomandy/2020-04-28-haiku_activity_report_april_2020.md:119]:
```suggestion
BeOS), but in others, we decided to follow what BeOS did.
```

File: content/blog/pulkomandy/2020-04-28-haiku_activity_report_april_2020.md (hunk 1 above)
Comment (waddlesplash) [reply content/blog/pulkomandy/2020-04-28-haiku_activity_report_april_2020.md:31]: Those are 3 dots. :)

Comment (humdingerb) [reply content/blog/pulkomandy/2020-04-28-haiku_activity_report_april_2020.md:31]: It's the UTF elipsis character that's still displayed (in Otter and Web+ at least) as "& mldr". Three separate dot characters would work around that. The best solution would of course be to make the website correctly display the elipsis...

Comment (waddlesplash) [reply content/blog/pulkomandy/2020-04-28-haiku_activity_report_april_2020.md:31]: No, I mean that the source for this post is already using 3 dots on line 31 above.

Comment (humdingerb) [reply content/blog/pulkomandy/2020-04-28-haiku_activity_report_april_2020.md:31]: Weird. It's still "& mldr;" on the site... Some 'helpful' auto-conversion somewhere?
